siwe_test.go

```diff
@@ -281,7 +332,7 @@ func parsingPositive(t *testing.T, cases map[string]interface{}) {
 	}
 }
 
-func validationNegative(t *testing.T, cases map[string]interface{}) {
+func verificationNegative(t *testing.T, cases map[string]interface{}) {
```

Comment:
more like a general question and not related for this PR but would it make sense to move those tests to the global test vectors as well at some point?